modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

```diff
@@ -41,7 +41,7 @@ Write a short introductory paragraph that provides an overview of the module. Pr
 
 . Use an unnumbered bullet (*) if the procedure includes only one step.
 
-.Verification steps
+.Verification
```

Comment:
For issue #147  @jherman, see here.

Reply:
I think you want @jherrman instead.

Reply:
Perfect, thanks!

Reply:
Thanks @VladimirSlavik Sorry, it wasn't populating Jiri's git name because I guess he isn't in the review group or something. Thanks for the correction!

@jherrman, thanks.